src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { focusElement } from '../../../common/utils/helpers';
+import OMBInfo from '../../../common/components/OMBInfo';
+import FormTitle from '../../../common/schemaform/components/FormTitle';
+
+import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+
+class IntroductionPage extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="This page needs attention"/>
+        <p>Equal to VA Form 21-526EZ.</p>
+        <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          startText="Start the Education Application"
```

Comment:
I assume you're leaving these edu reference in here to be addressed later, but pointing it out just in case.

Reply:
Yup. Exactly that.